comms/src/connectivity/requester.rs

```diff
@@ -54,6 +54,7 @@ pub enum ConnectivityEvent {
     PeerConnected(PeerConnection),
     PeerConnectFailed(NodeId),
     PeerBanned(NodeId),
+    ExcludePeerBan(NodeId),
```

Comment:
Is there something that subscribes to this event? I don't know if it's worth publishing it

Reply:
It looks like this is being used to communicate to the DHT Connectivity Actor that doesn't have a direct handle because it doesn't provide a service API. However, I don't think this is really the ideal way to plumb this communication in. While it might make sense to add a peer to the Allowlist via the Comms::connectivity manager how do you manage that list afterwards in terms of seeing who is on it and removing peers from it?